### PR TITLE
Update Atlassian Jira Readme

### PR DESCRIPTION
**Pull request by ellis-elastic**
Added link to Atlassian documentation

Type of change

- Enhancement

## What does this PR do?

Added link to Atlassian documentation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/atlassian_jira/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.2"
+  changes:
+    - description: Updarte Readme
```
Comment (andrewkroh) [inline packages/atlassian_jira/changelog.yml:4]: Typo

```suggestion
    - description: Update Readme
```

**Review by r00tu53r — COMMENTED** [review]
The deletions indicated in `docs/README.md` for field docs (`host.os.name.text ..... user.target.name.text`) shouldn't be here. 

I think its because the updated `docs/README.md` has not been committed/pushed. Please apply these steps for all the open PRs.

- Change readme under `_dev/build/docs/README.md` (if it exists)
- Run `elastic-package build`
- Commit `_dev/build/docs/README.md` and `docs/README.md`
- Push changes

File: packages/atlassian_jira/changelog.yml (hunk 1 above)
Comment (ellis-elastic) [reply packages/atlassian_jira/changelog.yml:4]: Fixed

**Comment by ellis-elastic** [comment]
@r00tu53r r00tu53r I did run build on the files.

**Comment by ellis-elastic** [comment]
I've added a space to the readme file so I could re run build

**Comment by r00tu53r** [comment]
> I've added a space to the readme file so I could re run build

@ellis-elastic there is no need to add space or make a spurious change to trigger the build. I suspect that your `elastic-package` version is old or the cache (under `~/.elastic-package`) is not ok. I've pushed the changes to this one to fix the problem. 

You could try this on the next/other PR -

- Remove the cache
`rm -rf ~/.elastic-package`

- Rebuild
`elastic-package build`

File: packages/atlassian_jira/_dev/build/docs/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Atlassian Jira Integration
 
-The Jira integration collects audit logs from the audit log files or the audit API.
+The Jira integration collects audit logs from the audit log files or the [audit API](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-audit-records/).
```
Comment (r00tu53r) [inline packages/atlassian_jira/_dev/build/docs/README.md:3]: I think the link mentioned here is not what package uses. Please use this link -

https://confluence.atlassian.com/jiracore/audit-log-improvements-for-developers-1019401815.html

Comment (ellis-elastic) [reply packages/atlassian_jira/_dev/build/docs/README.md]: done

**Review by r00tu53r — APPROVED** [review]
LGTM

**Pull request merged**